test/browser/fragments.test.js

```diff
@@ -739,29 +735,30 @@ describe('Fragment', () => {
 		clearLog();
 		render(<Foo condition={false} />, scratch);
 
-		expect(ops).to.deep.equal([]);
+		expect(ops).to.deep.equal([]); // Component should not have updated (empty op log)
 		expect(scratch.innerHTML).to.equal(html);
 		expectDomLogToBe([
-			'<div>1Hello1.insertBefore(<span>1, <span>1)',
```

Comment:
This original dom ops were just wrong. They don't match what the test was doing.